Zend/tests/gh15976/alias-names.phpt

```diff
@@ -0,0 +1,13 @@
+--TEST--
+GH-15976: Bad class alias names
+--FILE--
+<?php
+
+class_alias( 'stdClass', '_' );
+class_alias( 'stdClass', 'bool' );
+
+?>
+--EXPECTF--
+Deprecated: Using "_" as a type alias is deprecated since 8.4 in %salias-names.php on line 3
+
+Fatal error: Cannot use 'bool' as a type alias as it is reserved in %salias-names.php on line 4
```

Comment:
Ditto, this should be double quotes instead of single quotes:
```suggestion
Fatal error: Cannot use "bool" as a type alias as it is reserved in %salias-names.php on line 4
```